### PR TITLE
Home

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added a Home page of the mini  website

Live Link: https://ng-mini-website.netlify.app/home/

### CODE REVIEW

File: home/styles.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    .container{
+        width: 50%;
+        height: 100%;
+    }
+    .start-quiz{
+        width: 50%;
+        height: 20%;
+    }
+}
+:root{
```
Comment (akash-1318) [inline home/styles.css:12]: Add variables in Utility files.

**Review by akash-1318 — COMMENTED** [review]
Added one suggestion else everything looks good from my end.

**Review by ark2002 — APPROVED** [review]
apart from the changes suggested everything looks good

File: home/styles.css (hunk 1 above)
Comment (poojakelkar) [reply home/styles.css:12]: sure Thanks for PR

**Pull request merged**